CMakeLists.txt

```diff
@@ -44,6 +45,17 @@ if(NOT DEFINED ONNX_VERIFY_PROTO3)
   option(ONNX_VERIFY_PROTO3 "Generate code by proto3" ${PROTO3_ENABLED})
 endif()
 
+ if (ONNX_USE_PROTOBUF_SHARED_LIBS)
+  if(MSVC)
+    add_definitions(-DPROTOBUF_USE_DLLS)
+  else()
+    message(
```

Comment:
Would be better if you can check if ONNX_USE_MSVC_STATIC_RUNTIME is OFF.

Reply:
I dont quite understand... what purpose will this check serve?

Reply:
yes added the check... For current versions of protobuf from conda-forge which are built as a shared lib "ONNX_USE_MSVC_STATIC_RUNTIME" needs to be turned OFF